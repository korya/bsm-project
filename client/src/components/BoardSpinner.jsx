import React from 'react';
import Spinner from './Spinner';
import './Board.css';

export default function BoardSpinner() {
  return (
    <div className="sudoku-board-empty">
      <Spinner />
    </div>
  );
}
