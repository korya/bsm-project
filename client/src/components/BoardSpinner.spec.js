import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import BoardSpinner from './BoardSpinner';
import Spinner from './Spinner';

test('renders correctly', () => {
  expect(
    renderer.create(<BoardSpinner />).toJSON(),
  ).toMatchSnapshot();
});

test('renders Spinner', () => {
  const wrapper = shallow(<BoardSpinner />);
  expect(wrapper.find(Spinner)).toBeDefined();
});
