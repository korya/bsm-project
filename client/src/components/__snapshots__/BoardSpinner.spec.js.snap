// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sudoku-board-empty"
>
  <div
    className="spinner"
  />
</div>
`;
